doc/reference/tooling/tt_cli/export.rst

```diff
@@ -89,15 +104,62 @@ In this case, field values start from the second row, for example:
 Exporting compound data
 -----------------------
 
-By default, ``tt`` exports empty values for fields containing compound data such as arrays or maps.
+In the CSV format, ``tt`` exports empty values by default for fields containing compound data such as arrays or maps.
 To export compound values in a specific format, use the ``--compound-value-format`` option.
-For example, the command below exports compound values serialized in JSON:
+For example, the command below exports compound values to CSV serialized in JSON:
 
 .. code-block:: console
 
     $ tt crud export localhost:3301 customers:customers.csv  \
                      --compound-value-format json
 
+.. _tt-export-tdg2:
+
+Exporting from Tarantool Data Grid 2
+------------------------------------
+
+The command below exports data of the ``customers`` space from a TDG2 cluster to
```

Comment:
AFAIR, TDG2 does not uses the term "space" referring to its data storage structures. If I got it right, they use "type" in this case.